pg_search/src/postgres/storage/utils.rs

```diff
@@ -265,45 +282,64 @@ impl Drop for BM25BufferCache {
 
 type NeedsLock = bool;
 pub struct BufferMutVec {
-    inner: Vec<(pg_sys::Buffer, NeedsLock)>,
+    inner: [(pg_sys::Buffer, NeedsLock); MAX_BUFFERS_TO_EXTEND_BY],
+    cursor: usize,
```

Comment:
Please add a field doc for this field, or alternatively a struct doc for this struct.